src/fragments/lib/auth/ios/signin/20_confirmSignUp.mdx

```diff
@@ -43,3 +43,9 @@ func confirmSignUp(for username: String, with confirmationCode: String) -> AnyCa
 </Block>
 
 </BlockSwitcher>
+
+You will know the sign up flow is complete if you see the following in your console window:
```

Comment:
Note: This language was preexisting. It was moved from the shared fragments to allow flutter to omit this.